eclair-core/src/main/scala/fr/acinq/eclair/db/FileBackupHandler.scala

```diff
@@ -65,6 +66,8 @@ class FileBackupHandler private(databases: FileBackup, backupFile: File, backupS
 
       // publish a notification that we have updated our backup
       context.system.eventStream.publish(BackupCompleted)
+      Metrics.FileBackupCompleted.withoutTags().increment()
+      Metrics.FileBackupDuration.withoutTags().record(end - start)
```

Comment:
Why don't you use `KamonExt.time`?

Reply:
Done in https://github.com/ACINQ/eclair/pull/1475/commits/84aee65057bf3788b58b1ddae8dd4d0078f3a4ab